### PR TITLE
ai-kit: Some minor updates

### PR DESCRIPTION
**Pull request by naushir**
- Use sudo apt full-upgrade instead of sudo apt upgrade
- De-emphasise the optional part of the PCIe gen 3 switch
- Remove mention about custom nn models for now.

### CODE REVIEW

**Comment by naushir** [comment]
@nathan-contino some minor tweaks to the AI-kit docs.

File: documentation/asciidoc/accessories/ai-kit/getting-started.adoc [hunk 1]
```diff
@@ -19,7 +19,7 @@ For this guide, you will need the following:
 
 . Follow the xref:ai-kit.adoc#ai-kit-installation[installation instructions] to get your AI Kit hardware connected to your Raspberry Pi 5.
 
-. (optional) Follow the instructions to xref:../computers/raspberry-pi.adoc#pcie-gen-3-0[enable PCIe Gen 3.0]. This helps achieve the best performance with your AI Kit.
+. Follow the instructions to xref:../computers/raspberry-pi.adoc#pcie-gen-3-0[enable PCIe Gen 3.0]. This step is optional, but HIGHLY RECOMMENDED to achieve the best performance with your AI Kit.
```
Comment (nathan-contino) [inline documentation/asciidoc/accessories/ai-kit/getting-started.adoc:22]:
```suggestion
. Follow the instructions to xref:../computers/raspberry-pi.adoc#pcie-gen-3-0[enable PCIe Gen 3.0]. This step is optional, but _highly recommended_ to achieve the best performance with your AI Kit.
```

**Review by nathan-contino — APPROVED** [review]
LGTM, thanks! Made one change to better align with the rest of the docs (in general, we prefer emphasis to caps-case), but otherwise I like all of these changes.

**Comment by naushir** [comment]
Excellent.  Feel free to merge when you are ready.  I assume this will then make it to the main site onced merged into develop?

**Comment by nathan-contino** [comment]
Nah, we only have auto-deploy set up for `master`. However, I just opened a PR to deploy this content to production (via master merge) (https://github.com/raspberrypi/documentation/pull/3689)[here]! I'll merge it in as soon as it passes checks, should be live in the next 10 minutes or so. Thanks especially for catching the `full-upgrade` issue -- I should have noticed that!

**Pull request merged**